src/components/QRCodeCameraPickerCanvas.js

```diff
@@ -152,10 +213,10 @@ export default class QRCodeCameraPickerCanvas extends Component<
             this.unsubscribes.push(() => cancelAnimationFrame(raf))
           }
         })
-        .catch(e => {
+        .catch(_ => {
           if (this.unmounted) return
           this.setState({
-            message: String(e.message || e.name || e),
+            message: <Trans i18nKey="errors.NoAccessToCamera.description" />,
```

Comment:
@juan-cortes I would prefer we have a new `NoAccessToCamera` error, we `setState({ error: new NoAccessToCamera(e.message) })` and we use `<TranslatedError` in the rendering part. it's the general convention we use everywhere.

Reply:
Sure.... I'll make a pr on common to add the error